results/cmd/api/main.go

```diff
@@ -57,7 +65,27 @@ type server struct {
 	db *sql.DB
 }
 
+// Receive TaskRun from Watcher and save it to local Sqlite Server
```

Comment:
small nitpick: The comment should start with the name of the function i.e. 
`CreateTaskRun receives TaskRuns from Watcher...`

See https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
done